README.md

```diff
@@ -209,6 +209,16 @@ This works on Linux and MacOS.
 cmake -DCMAKE_EXPORT_COMPILE_COMMANDS=1 ...
 ```
 
+#### ... with `meson`
+
+When creating the initial build directory with `meson`,
+it will automatically generate a `compile_commands.json`
+file inside of `builddir`.
+
+```sh
+meson setup builddir
```

Comment:
Could you make it clearer that `builddir` is an arbitrary name?  `$build_dir` or `<build_dir>` would be good.

Reply:
For both `builddir`s.

Reply:
Oh yes, my bad.